app/views/user_mentee_applications/mentee_application_states/_withdrawn.html.erb

```diff
@@ -0,0 +1,6 @@
+<h3>Withdrawn</h3>
+
+<p>You have withdrawn your application to the Agency.</p>
+
+<!-- TODO the notion references links? What might those be? -->
```

Comment:
Notion references different links that we could give the user here. What did people have in mind for this?